examples/odata-cli/src/activities.ts

```diff
@@ -210,6 +211,10 @@ export async function testUiServiceGenerator(
     const bo = bos.find((bo) => bo.name === env.TEST_BO_NAME);
     logger.debug(bos.map((bo) => bo.name));
 
+    // Get Abap CDS Views
```

Comment:
Do we need this commented out?

Reply:
This is used for manually testing, so I've added this blocked commented as it should not be run together with business object search and can be uncommented and used for testing when needed.

Reply:
Can you add a comment to indicate thats why this is here?